site/content/docs/developing/storage.md

```diff
@@ -30,6 +30,16 @@ $ copilot storage init -n users -t DynamoDB -w api --partition-key id:N --sort-k
 
 This will create a DynamoDB table called `${app}-${env}-${svc}-users`. Its partition key will be `id`, a `Number` attribute; its sort key will be `email`, a `String` attribute; and it will have a [local secondary index](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/LSI.html) (essentially an alternate sort key) on the `Number` attribute `post-count`. 
 
+It is also possible to create an RDS Aurora Serverless(Daiya: add link here mmkay?) cluster using `copilot storage init`. 
+```bash
+# For a guided experience.
+$ copilot storage init -t RDS
+
+# Or by providing flags to skip the prompts.
+$ copilot storage init -n my-cluster -t Aurora -w api --engine PostgreSQL --initial-db my_db
+```
+This will create an RDS Aurora Serverless cluster that uses PostgreSQL engine with a database named `my_db`. An environment variable named `MYCLUSTER_SECRET` is injected into your workload as a JSON string. The fields are `'host'`, `'port'`, `'dbname'`, `'username'`, `'password'`, `'dbClusterIdentifier'`, `'engine'`.
```

Comment:
```suggestion
This will create an RDS Aurora Serverless cluster that uses the PostgreSQL engine with a database named `my_db`. An environment variable named `MYCLUSTER_SECRET` is injected into your workload as a JSON string. The fields are `'host'`, `'port'`, `'dbname'`, `'username'`, `'password'`, `'dbClusterIdentifier'`, and `'engine'`.
```